internal/fs/local_file_test.go

```diff
@@ -427,3 +428,49 @@ func (t *LocalFileTest) TestRecursiveListingWithLocalFiles() {
 	t.closeFileAndValidateObjectContents(&t.f2, "explicitFoo/"+explicitLocalFileName, "")
 	t.closeFileAndValidateObjectContents(&t.f3, ""+FileName, "")
 }
+
+func (t *LocalFileTest) TestLocalFileRenameFails() {
```

Comment:
nit: Test should be following by name of the method being tested.
TestRenameOfLocalFileFails.